### DISCUSSION TITLE
Planned features / capabilities / roadmap

### DISCUSSION THREAD

**Discussion by MHHenriksen** [community/community#3531] [Codespaces] [answered]
Hey, is there an overview of what you're working on anywhere? And what's planned for launch and what comes later?

**Comment by PranamBhat** [comment, marked as answer]
Hey @MHHenriksen 

You can always check here : [https://github.com/github/roadmap/projects/1](https://github.com/github/roadmap/projects/1)

**Comment by 2percentsilk** [comment]
We don't have a specific Codespaces roadmap to share currently but the link @PranamBhat provided (https://github.com/github/roadmap/projects/1) gives you a more general timeline 😄 

If you have specific feature requests, feel free to log them in this discussions category as we'll be using this space as a way to gauge interest in certain features and provide updates/ETA as available!
